code/modules/mapping/mapping_helpers.dm

```diff
@@ -1183,6 +1183,10 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 	icon_state = "damaged_window"
 	layer = ABOVE_OBJ_LAYER
 	late = TRUE
+	/// Minimum roll of integrity damage in percents
+	var/integrity_min_factor = 0.2
+	/// Maximum roll of integrity damage in percents
+	var/integrity_max_factor = 0.8
```

Comment:
no i meant like real and proper `#define`s since this doesn't get mutated anywhere but i suppose this is okay in the event that someone wants to make a subtype that does more damage / less damage.

Reply:
(no change)

Reply:
Oh, okay... Thanks!